### ISSUE TITLE
webpage update in server but not updated client side app 

### ISSUE THREAD

**Issue by SachinTichkule** [gree/unity-webview#1049]
need to update the webview then i have to clear my app data then changes are seen in our webview in unity app.
what should i do to prevent this issue.

**Comment by KojiNakamaru** [comment]
The plugin has `webViewObject.ClearCache(true)` for Android/iOS. For any platform, adding a url parameter is a most reliable way:

```c#
        webViewObject.LoadURL(Url.Replace(" ", "%20") + "?_=" + (System.DateTime.UtcNow.Subtract(new System.DateTime(1970, 1, 1))).TotalSeconds);
```

You can also utilize the following to forcibly reload the current page.

```c#
        webViewObject.EvaluateJS("window.location.href = window.location.href.replace(/#.*$/, '')")
```

**Comment by SachinTichkule** [comment]
webViewObject.ClearCache(true);
it take little long time to load url in webview means its shows black screen for long time.

**Comment by KojiNakamaru** [comment]
Yes, so it is better to utilize other methods. If you control the html server, you can also disable cache by apache .htaccess, for example (cf. https://stackoverflow.com/questions/22891965/how-to-disable-cache-of-apache/22892020#22892020 ). Such settings should be okay at least for debugging.
